lcls_tools/common/pyepics_tools/pyepicsUtils.py

```diff
@@ -0,0 +1,4 @@
+EPICS_NO_ALARM_VAL = 0
```

Comment:
What are these for? I'm not familiar.

Reply:
It's for checking a PV's severity (like, whether the PV is red or yellow or purple)